es-core/src/Window.cpp

```diff
@@ -335,8 +335,8 @@ void Window::setHelpPrompts(const std::vector<HelpPrompt>& prompts, const HelpSt
 				// yes, it has!
 
 				// can we combine? (dpad only)
-				if((it->first == "up/down" && addPrompts.at(mappedTo->second).first == "left/right") ||
-					(it->first == "left/right" && addPrompts.at(mappedTo->second).first == "up/down"))
+				if((strcmp(it->first, "up/down") == 0 && addPrompts.at(mappedTo->second).first == "left/right") ||
```

Comment:
I was looking into this exact same set of changes.

It seems the HelpPrompts are a pair of standard C strings (const char*).

Read: https://www.daniweb.com/programming/software-development/threads/78711/why-strcmp-when-can-use-operator

As such, I did not object to the changes here. I know C is a lot trickier in this kind of things, and I am far from up to date on that.